common/special_troops/wc_special_troops.txt

```diff
@@ -106,7 +106,7 @@ insectoidguard = {
 	base_type = heavy_infantry
 }
 
-demonicshock = {
```

Comment:
Demonic units are probably also broken due to being removed.

<details>
<summary>Click to expand</summary>

![ck2_1](https://user-images.githubusercontent.com/46576860/55299931-f1530980-543d-11e9-9e29-2dc2a98fa4d1.png)

</details>